x-pack/plugins/uptime/public/components/overview/query_bar/use_query_bar.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import React, { useCallback, useState } from 'react';
-import { useDebounce } from 'react-use';
+import useDebounce from 'react-use/lib/useDebounce';
```

Comment:
This PR fixes an eslint rule that should have been firing, but wasn't. I've updated this import to comply with the fixed rule.